packages/graphql-armor/src/apollo/armor.ts

```diff
@@ -48,3 +56,9 @@ export class ApolloArmor {
     };
   }
 }
+
+export class ApolloV4Armor extends ApolloArmor {
+  constructor(config: GraphQLArmorConfig = {}, version = 'v4') {
+    super(config, version);
+  }
+}
```

Comment:
Hello @c3b5aw ,

You're right. Updated the context from `<{}>` to `<BaseContext>` and totally ditched the `ApolloServerPlugin` type and use `any` so we can just use `ApolloArmor` for both Apollo Server 3 and 4 projects instead of having an `ApolloV4Armor`.

Here: https://github.com/Escape-Technologies/graphql-armor/pull/354/commits/92218f2f5f17f1932ccfcdfcccb27914a3a3bfc2